pandas/tests/frame/test_arithmetic.py

```diff
@@ -1297,7 +1297,7 @@ def test_strings_to_numbers_comparisons_raises(self, compare_operators_no_eq_ne)
             f(df, 0)
 
     def test_comparison_protected_from_errstate(self):
-        missing_df = tm.makeDataFrame()
+        missing_df = tm.makeDataFrame()._consolidate()
```

Comment:
you are already doing this on the creation (I understand by find this fragile)

Reply:
> you are already doing this on the creation

i dont think so

> (I understand by find this fragile)

I agree.  Silver lining: finding the existing fragility.

Reply:
actuallly you *are* doing this on creation, maybe you recently added. prefer NOT to do this in the tests proper (in pandas/testing is ok)

Reply:
updated to do the consolidation in tm.makeDataFrame